RestaurantReservationSystem/EntityOperations/CustomerOperations.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db;
+using RestaurantReservation.Db.Models;
+public class CustomerOperations
```

Comment:
aren't all these operation classes act like repos? why not call them repos and move to DB layer? and i see duplicate classes like EmployeeRepository and EmployeeOperations, both query the employees table.